src/main/java/DukeException.java

```diff
@@ -0,0 +1,34 @@
+public class DukeException extends Exception {
```

Comment:
Maybe you should use subclasses of a DukeException to trigger different warning messages and also so that the subclasses could receive additional input such as task number if need be.
The idea of keeping the error messages here is nice however so that the same errors receive the same message.